generators/entity-client/templates/react/src/main/webapp/app/entities/entity.tsx.ejs

```diff
@@ -91,6 +93,11 @@ export class <%= entityReactName %> extends React.Component<I<%= entityReactName
             <FaPlus /> <Translate contentKey="<%= keyPrefix %>home.createLabel" />
           </Link>
         </h2>
+        {(updateFailure) ? (
```

Comment:
I was gonna add toaster for this so that error and notification is global like in Angular. So let me update this

Reply:
Alright